openmrs/src/main/java/org/motechproject/openmrs/service/impl/OpenMRSEncounterServiceImpl.java

```diff
@@ -180,18 +180,24 @@ private void validateEncounter(Encounter encounter) {
         List<Observation> updatedObs = new ArrayList<>();
         if (originalObservations != null) {
             for (Observation observation : originalObservations) {
-                String conceptUuid = conceptService.resolveConceptUuidFromConceptName(config.getName(), observation.getConcept().getName().getName());
-                if (CollectionUtils.isNotEmpty(observation.getGroupsMembers())) {
-                    resolveConceptUuidForConceptNames(config, observation.getGroupsMembers());
+                if (observationValueIsNotEmpty(observation)) {
```

Comment:
additionally, please log info if value is empty, that we omit creating this observation because of empty value. I think WARN level will be ok